GameServerLib/Content/CharData.cs

```diff
@@ -110,7 +112,7 @@ public void Load(string name)
             HpRegenPerLevel = file.GetFloat("Data", "HPRegenPerLevel", HpRegenPerLevel);
             MpRegenPerLevel = file.GetFloat("Data", "MPRegenPerLevel", MpRegenPerLevel);
             AttackSpeedPerLevel = file.GetFloat("Data", "AttackSpeedPerLevel", AttackSpeedPerLevel);
-            IsMelee = file.GetString("Data", "IsMelee", IsMelee ? "Yes" : "No").Equals("yes");
+            IsMelee = file.GetString("Data", "IsMelee", IsMelee ? "true" : "false").Equals("true");
```

Comment:
Did we change something in the way we store attack data? Because in real inibins, the value is stored as "Yes" and "No".

Reply:
Json files in Content store it as true or false as far as I checked.

Reply:
Alright then.